profile.js

```diff
@@ -0,0 +1,39 @@
+const body = document.querySelector('[data-js="body"]');
+const card = document.querySelector('[data-js="card"]');
+const header = document.querySelector('[data-js="header"]');
+const footer = document.querySelector('[data-js="footer"]');
+const navItemSelected = document.querySelector(
+  '[data-js="nav__item-selected"]'
+);
+console.log(navItemSelected);
```

Comment:
We always want to delete code, we do not need before pushing to our remote repository. As this `console.log` is not needed for your app to function as it should, please remove it.